flink-dist/src/main/assemblies/bin.xml

```diff
@@ -84,6 +84,28 @@ under the License.
 			<fileMode>0644</fileMode>
 		</file>
 
+		<!-- Built-in formats -->
+		<file>
+			<source>../flink-formats/flink-csv/target/flink-csv-${project.version}.jar</source>
+			<outputDirectory>lib/</outputDirectory>
+			<destName>flink-csv-${project.version}.jar</destName>
```

Comment:
Not sure, but shall we use `flink-csv-xxx-sql-jar.jar`?

Reply:
I think we can respect module name.

Reply:
Shall we remove `sql-jar` since there is no difference between them so far, and it confuses users? cc @twalthr